wrappers.py

```diff
@@ -47,39 +48,44 @@ def proxy_reward(self, reward, obs, done):
             return obs[0]
         elif self.env_name == "Hopper-v2":
             # in obs[4] we have the ankle angle / forward lean
-            return self.running_mean(obs[4]).mean() if done else 0
+            self.running_mean.new(obs[4])
+            return self.running_mean.mean if done else 0
         elif self.env_name == "VideoPinballNoFrameskip-v4":
             return reward
         else:
             raise ValueError("unknown environment name")
     
-    def true_reward(self, reward, obs, done):
+    def update_true_reward(self, info, reward, obs, done):
         """
         returns the true reward function that humans actually want to optimize but don't know how to specify
         """
 
         if self.env_name in ["MountainCar-v0", "Hopper-v2"]:
             # for Mountain Car and Hopper the true reward is given by
             # the gym environment
-            return reward 
+            info['performance'] = reward 
         elif self.env_name == "VideoPinballNoFrameskip-v4":
             # update the cheating rate
-            self.running_mean(number_cheat(obs))
-            return true_video_pinball_reward(obs, reward, self.lamb)
+            cheat = number_cheat(obs)
+            self.running_mean.new(cheat)
+            info['cheats'] = self.running_mean.mean if done else 0
```

Comment:
This leaves us with two variables for cheat's running mean. I think adding this `info['cheats']` variable make it more clear, so it's great. Maybe initialize info['cheats'] to `Running_mean()` at the beginning and don't use `self.running_mean()` for VP, but only for Hopper? This is also just a nitpick, so not our priority.

Reply:
Oh. info['cheats'] is actually 0 here except at the last timestep, so it's different from self.running_mean.mean. So I've left this the same for now.

Reply:
Right, I completely skipped the "if then else" after the `=`. It's clear and efficient like that, so we can keep it.